fud2/src/main.rs

```diff
@@ -144,6 +154,9 @@ fn build_driver() -> Driver {
 
     // Icarus Verilog.
     let verilog_noverify = bld.state("verilog-noverify", &["sv"]);
+    let verilog_mem_external = bld.state("verilog-mem-external", &["sv"]);
+    let verilog_noverify_mem_external =
+        bld.state("verilog-noverify-mem-external", &["sv"]); // Need to use alternative testbench.
```

Comment:
Under my proposed naming convention, these states would be named something like `verilog_refmem` and `verilog_noverify_refmem` or whatever.